src/tiled/tilesetdocument.h

```diff
@@ -64,6 +63,8 @@ class TilesetDocument : public Document
     void setWriterFormat(TilesetFormat *format);
 
     QString displayName() const override;
+    TilesetFormat* exportFormat() const override;
+    void setExportFormat(FileFormat *format) override;
```

Comment:
Please move declarations up to avoid having unrelated function `displayName` in between.

Reply:
Done